pydoctor/epydoc/markup/_pyval_repr.py

```diff
@@ -198,7 +198,9 @@ def __init__(self, document: nodes.document, is_complete: bool, warnings: List[s
     def to_stan(self, docstring_linker: DocstringLinker) -> Tag:
         return Tag('code')(super().to_stan(docstring_linker))
 
-def colorize_pyval(pyval: Any, linelen:Optional[int], maxlines:int, linebreakok:bool=True, refmap:Optional[Dict[str, str]]=None) -> ColorizedPyvalRepr:
+def colorize_pyval(pyval: Any, linelen:Optional[int], maxlines:int, 
+                   linebreakok:bool=True, refmap:Optional[Dict[str, str]]=None, 
+                   is_annotation: bool = False) -> ColorizedPyvalRepr:
```

Comment:
should we be adopting Black so that this sort of style change doesn't require discretion? :)